src/main.rs

```diff
@@ -66,9 +66,12 @@ pub struct Luro {
     pub hyper: HyperClient<HttpConnector>,
     pub user: CurrentUser,
     pub standby: Standby,
-    pub commands: std::sync::RwLock<LuroCommands>,
+    pub commands: std::sync::RwLock<LuroCommands>, 
     pub guild_settings: tokio::sync::RwLock<LuroGuilds>,
     pub hecks: tokio::sync::RwLock<Hecks>,
+    
+    // Global Vars
+    interaction_count: Arc<tokio::sync::RwLock<u32>>,   
```

Comment:
Luro's context is already wrapped in an Arc, you don't need it here. My previous comment was more of a note about thread safety. That said, you should just need to accept this suggestion and be fine :)

```suggestion
    interaction_count: tokio::sync::RwLock<u32>,   
```